src/App.tsx

```diff
@@ -1,11 +1,62 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import * as todosApi from './api/todos';
+import { Todo } from './types/Todo';
+import cn from 'classnames';
+import { Status } from './types/Status';
+import { getFilterTodos } from './utils/getFilterTodos';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [titles, setTitles] = useState('');
+  const [errorMessage, setErrorMessage] = useState<string>('');
+  const [filter, setFilter] = useState<Status>(Status.All);
+
+  useEffect(() => {
+    todosApi
+      .getTodos()
+      .then(setTodos)
+      .catch(() => {
+        setErrorMessage(`Unable to load todos`);
+        setTimeout(() => setErrorMessage(''), 3000);
```

Comment:
Use this function to add delay.
<img width="415" alt="Screenshot 2024-03-31 at 09 18 16" src="https://github.com/mate-academy/react_todo-app-loading-todos/assets/14590796/72256c65-3847-480f-832c-13619ef08aec">